src/hooks/workspace/use-workspace-instruction-modal.ts

```
@@ -0,0 +1,267 @@
+    let isNewWorkspaceTrigger = false;
+    try {
+      isNewWorkspaceTrigger = newWorkspaceStorageKeys.some(
+        (storageKey) => window.localStorage.getItem(storageKey) === "true"
+      );
+    } catch {
+      isNewWorkspaceTrigger = false;
+    }
+
+    if (isNewWorkspaceTrigger) {
+      try {
+        newWorkspaceStorageKeys.forEach((storageKey) => window.localStorage.removeItem(storageKey));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)